anu_lms.routing.yml

```diff
@@ -0,0 +1,8 @@
+ anu_lms.course.finish:
+  path: '/anu-lms/finish/{node}'
```

Comment:
How about just `/node/{node}/finish` ? Can be better semantically

Reply:
I agree. Changed